packages/visual-editing/src/ui/telemetry/sendTelemetry.ts

```diff
@@ -0,0 +1,34 @@
+import {defineEvent} from '@sanity/telemetry'
+import type {VisualEditingNode} from '../../types'
+
+const events = {
+  'Visual Editing Drag Sequence Complete': defineEvent({
+    name: 'Visual Editing Drag Sequence Complete',
+    description: 'An array is successfully reordered using drag and drop.',
+    version: 1,
+  }),
+}
+
+type EventDataMap = {
+  [K in keyof typeof events]: (typeof events)[K] extends ReturnType<typeof defineEvent<infer T>>
+    ? T
+    : never
+}
+
+function sendTelemetry<K extends keyof typeof events>(
```

Comment:
I debated doing this a bunch of ways, but as this is an early iteration I feel like a simple function is a safe bet. We can expand as we need?

Reply:
Yep I think it's fine 👍

Reply:
Excellent :)